crates/matrix-sdk-crypto/src/store/integration_tests.rs

```diff
@@ -855,12 +856,15 @@ macro_rules! cryptostore_integration_tests {
                 let settings_1 = RoomSettings {
                     algorithm: EventEncryptionAlgorithm::MegolmV1AesSha2,
                     only_allow_trusted_devices: true,
+                    session_rotation_period: Some(Duration::from_millis(10000)),
```

Comment:
It's probably a bit more Rust idiomatic to write this as follows:

```diff
- Duration::from_millis(10000)),
+ Duration::from_millis(10_000)),
```

I also wonder if we could simply use : `from_secs(10)`?

Reply:
sure